src/web/routes/application/middleware/sanitize.js

```diff
@@ -1,13 +1,16 @@
 const { map, trim } = require('ramda')
+const escapeHtml = require('escape-html')
 
 const trimValues = map(trim)
+const escapeValues = map(escapeHtml)
 
 const sanitize = (req, res, next) => {
-  req.body = trimValues(req.body)
+  req.body = escapeValues(trimValues(req.body))
```

Comment:
I think this line is mapping over all input values twice. A more efficient way to write this would be something like:
```
const sanitiseInput = compose(escapeHtml, trim)
const sanitiseBody = map(sanitiseBody)
req.body = sanitiseBody(req.body)
```